sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedGranularityType.cs

```diff
@@ -5,8 +5,51 @@
 
 namespace Azure.AI.MetricsAdvisor.Models
 {
+    /// <summary>
+    /// The type of <see cref="DataFeedGranularity"/>.
+    /// </summary>
     [CodeGenModel("Granularity")]
     public readonly partial struct DataFeedGranularityType
```

Comment:
This feels more like the frequency type.  Could this be consider for renaming?

Reply:
Added to the "design considerations" issue.
> Is `Granularity` a term often used to indicate frequency? If not, we should consider renaming it.